linux/NvidiaJetson.c

```diff
@@ -0,0 +1,321 @@
+#include "config.h"
+#ifdef NVIDIA_JETSON
+
+#include <ctype.h>
+#include <dirent.h>
+#include <stdio.h>
+#include <string.h>
+
+#include "CRT.h"
+#include "Hashtable.h"
+#include "Macros.h"
+#include "Meter.h"
+#include "XUtils.h"
+
+#include "linux/NvidiaJetson.h"
+#include "linux/LinuxProcess.h"
```

Comment:
Missed that ordering issue …

```suggestion
#include "linux/LinuxProcess.h"
#include "linux/NvidiaJetson.h"
```